akka-actor/src/main/scala/akka/actor/ActorPath.scala

```diff
@@ -254,6 +254,10 @@ sealed trait ActorPath extends Comparable[ActorPath] with Serializable {
  */
 @SerialVersionUID(1L)
 final case class RootActorPath(address: Address, name: String = "/") extends ActorPath {
+  require(name.length == 1 || name.indexOf('/', 1) == -1,
```

Comment:
- note in migration guide might not hurt, sure;
- nope, you can't - `copy` calls the constructor - thus the `require` will catch such attempts